CMakeLists.txt

```diff
@@ -831,9 +831,8 @@ if(NOT CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
 endif()
 
 if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
-  target_compile_options(libdevilutionx PUBLIC "/W1")
+  target_compile_options(libdevilutionx PUBLIC "/W2")
   target_compile_options(libdevilutionx PUBLIC "/Zc:__cplusplus")
-  target_compile_options(libdevilutionx PUBLIC "/permissive-")

```

Comment:
Would be nice to not conflate the warning level change with this completely unrelated warning.